.gitignore

```diff
@@ -1,3 +1,5 @@
 /gh-pages/
 tmp*
 ct_previous*
+# ignore all admin.conf files
```

Comment:
probably don't need this comment as its in the `gitignore` file

Reply:
ha, good point.